File: Servidor/Transaccional/Comandos/Comandos/ComandosUsuario/ComandoConsultarUsuarioPorID.cs
using System;
using Trascend.Bolet.ObjetosComunes.Entidades;

namespace Trascend.Bolet.Comandos.Comandos.ComandosUsuario
{
    public class ComandoConsultarUsuarioPorID : ComandoBase<Usuario>
    {
        Usuario _usuario;

        /// <summary>
        /// Constructor predeterminado
        /// </summary>
        /// <param name="usuario">Usuario a consultar</param>
        public ComandoConsultarUsuarioPorID(Usuario usuario)
        {
            this._usuario = usuario;
        }

        /// <summary>
        /// Método que ejecuta el comando
        /// </summary>
        public override void Ejecutar()
        {
            throw new NotImplementedException();
        }
    }
}

File: src/Servidor/Transaccional/Comandos_Fac/Comandos/ComandosUsuario/ComandoConsultarUsuarioPorID.vb
Imports Diginsoft.Bolet.ObjetosComunes.Entidades

Namespace Comandos.ComandosUsuario
    Public Class ComandoConsultarUsuarioPorID
        Inherits ComandoBase(Of Usuario)
        Private _usuario As Usuario

        ''' <summary>
        ''' Constructor predeterminado
        ''' </summary>
        ''' <param name="usuario">Usuario a consultar</param>
        Public Sub New(ByVal usuario As Usuario)
            Me._usuario = usuario
        End Sub

        ''' <summary>
        ''' Método que ejecuta el comando
        ''' </summary>
        Public Overrides Sub Ejecutar()
            Throw New NotImplementedException()
        End Sub
    End Class
End Namespace